src/MongoDB/Monitoring/functions.c

```diff
@@ -25,77 +25,50 @@
 
 ZEND_EXTERN_MODULE_GLOBALS(mongodb)
 
-static char* php_phongo_make_subscriber_hash(zval* subscriber)
-{
-	char* hash;
-	int   hash_len;
-
-	hash_len = spprintf(&hash, 0, "SUBS-%09d", Z_OBJ_HANDLE_P(subscriber));
```

Comment:
After reading through SplObjectStorage's implementation, I realized [`spl_object_storage_get_hash()`](https://github.com/php/php-src/blob/php-8.0.3/ext/spl/spl_observer.c#L78) just uses the object's handle as a numeric hash key. I don't recall why we originally had this logic to use a string key, but it seems unnecessary.